elixir/apps/domain/lib/domain/policies/policy.ex

```diff
@@ -4,8 +4,8 @@ defmodule Domain.Policies.Policy do
   schema "policies" do
     field :description, :string
 
-    belongs_to :actor_group, Domain.Actors.Group, where: [deleted_at: nil]
```

Comment:
This caused issues with preloads: we tried to fetch policy assocs to build it's name but they were not loaded due to filter. If we load them the link will go to 404 though, that's where this issue started from.